proto/dymensionxyz/dymension/incentives/params.proto

```diff
@@ -29,7 +29,4 @@ message Params {
     (gogoproto.nullable) = false,
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int"
   ];
-  // FeatureFlagEpochEndDistribution enables distribution of gauge rewards
-  // at the end of each epoch. If disabled, rewards are not distributed.
-  bool feature_flag_epoch_end_distribution = 5;
```

Comment:
this flag already on Playground, remove like this won't cause trouble?

Reply:
it won't in theory. the first 4 fields are unchanged, so there should not be any serialization issues.